it/test/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositoryISpec.scala

```diff
@@ -3561,17 +3560,17 @@ class ApplicationRepositoryISpec
   }
 
   def anApplicationDataForTest(
-      id: ApplicationId,
-      prodClientId: ClientId = ClientId("aaa"),
-      state: ApplicationState = testingState(),
-      access: Access = Access.Standard(),
-      grantLength: Int = defaultGrantLength,
-      users: Set[Collaborator] = Set(
+                                id: ApplicationId,
```

Comment:
This new indentation seems wrong. Have you run scalafmt/scalafix (e.g. run_all_tests.sh)?

Reply:
I found the problem. Please change the last two lines of build.sbt to
```
  Command.command("clean-and-test") { state => "cleanAll" :: "compile" :: "run-all-tests" :: state },
  Command.command("pre-commit") { state => "cleanAll" :: "fmtAll" :: "fixAll" :: "coverage" :: "testAll" :: "coverageOff" :: "coverageAggregate" :: state }
```

Earlier changes to build.sbt were not the latest update, so the formatting is not happening in all subprojects.